tests/ieee802154_hal/main.c

```diff
@@ -78,6 +115,7 @@ static int print_addr(int argc, char **argv)
         printf("%02x", *_p++);
     }
     printf("\n");
+    printf("%d", current_channel);
```

Comment:
If this is going to print the channel, it needs a text (e.g "Channel: "). Otherwise it should be removed.